playbooks/byo/config.yml

```diff
@@ -4,3 +4,9 @@
 
 - name: Run the openshift-node config playbook
   include: openshift-node/config.yml
+
+- name: Run the etcd config playbook
+  include: etcd/config.yml
```

Comment:
This is going to get a bit sticky I think...  We'll need to make sure that we still support embedded etcd for the single master config.

At bare minimum, I think we'll need to create a variable that we can use as a conditional on the etcd includes, we could use the same one to disable the embedded etcd on the masters as well.

The other thing I see of the bat is that we'll want to configure the etcd cluster before we configure the masters and the nodes.